doc/HD44780.rst

```diff
@@ -99,12 +110,12 @@ Here is a small example of how this can be leveraged.
 .. code:: python
 
 
-  from luma.core.interface.serial import parallel
+  from luma.core.interface.parallel import bitbang_6800
   from luma.lcd.device import hd44780
   from luma.core.render import canvas
   from PIL import Image, ImageDraw
 
-  interface = parallel(RS=7, E=8, PINS=[25,24,23,27])
+  interface = bitbang_6800(RS=7, E=8, PINS=[25,24,23,27])
```

Comment:
add proper spaces between items in `PINS` list

Reply:
Changed as suggested